### PR TITLE
Lighttpd rewording, remove PHP 5 refs

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: install/unix/lighttpd-14.xml [hunk 1]
```diff
@@ -14,19 +14,16 @@
...
+  automagically enabled in php-cgi.
  </para>
  
  <sect2 xml:id="install.unix.lighttpd-14.lighttpd-spawn">
   <title>Letting Lighttpd spawn php processes</title>
 
   <para>
-   To configure Lighttpd to connect to php and spawn fastcgi processes, edit
-   lighttpd.conf. Sockets are preferred to connect to fastcgi processes on
+   To configure Lighttpd to connect to php and spawn FastCGI processes, edit
```
Comment (cmb69) [inline install/unix/lighttpd-14.xml:25]: Nit: either
```suggestion
   To configure Lighttpd to connect to PHP and spawn FastCGI processes, edit
```
or
```suggestion
   To configure Lighttpd to connect to php-cgi and spawn FastCGI processes, edit
```

File: install/unix/lighttpd-14.xml [hunk 2]
```diff
@@ -14,19 +14,16 @@
...
+   <literal>lighttpd.conf</literal>. Sockets are preferred to connect to FastCGI processes on
```
Comment (cmb69) [inline install/unix/lighttpd-14.xml:26]: Maybe use
```suggestion
   <filename>lighttpd.conf</filename>. Sockets are preferred to connect to FastCGI processes on
```

File: install/unix/lighttpd-14.xml [hunk 3]
```diff
@@ -55,15 +52,15 @@ fastcgi.server = ( ".php" =>
   </example>
 
   <para>
-   The bin-path directive allows lighttpd to spawn fastcgi processes dynamically.
-   PHP will spawn children according to the PHP_FCGI_CHILDREN environment
-   variable.  The "bin-environment" directive sets the environment for the
+   The <literal>bin-path</literal> directive allows lighttpd to spawn fastcgi processes dynamically.
+   PHP will spawn children according to the <varname>PHP_FCGI_CHILDREN</varname> environment
+   variable. The <literal>bin-environment</literal> directive sets the environment for the
```
Comment (cmb69) [inline install/unix/lighttpd-14.xml:57]:
```suggestion
   The <filename>bin-path</filename> directive allows lighttpd to spawn FastCGI processes dynamically.
   PHP will spawn children according to the <envar>PHP_FCGI_CHILDREN</envar> environment
   variable. The <literal>bin-environment</literal> directive sets the environment for the
```

File: install/unix/lighttpd-14.xml [hunk 4]
```diff
@@ -55,15 +52,15 @@ fastcgi.server = ( ".php" =>
...
    spawned processes. PHP will kill a child process after the number of
-   requests specified by PHP_FCGI_MAX_REQUESTS is reached. The directives
-   "min-procs" and "max-procs" should generally be avoided with PHP. PHP
+   requests specified by <varname>PHP_FCGI_MAX_REQUESTS</varname> is reached. The directives
```
Comment (cmb69) [inline install/unix/lighttpd-14.xml:59]:
```suggestion
   requests specified by <envar>PHP_FCGI_MAX_REQUESTS</envar> is reached. The directives
```

File: install/unix/lighttpd-14.xml [hunk 5]
```diff
@@ -55,15 +52,15 @@ fastcgi.server = ( ".php" =>
...
+   <literal>min-procs</literal> and <literal>max-procs</literal> should generally be avoided with PHP. PHP
    manages its own children and opcode caches like APC will only share among
-   children managed by PHP. If "min-procs" is set to something greater than 1,
-   the total number of php responders will be multiplied PHP_FCGI_CHILDREN
+   children managed by PHP. If <literal>min-procs</literal> is set to something greater than <literal>1</literal>,
+   the total number of php responders will be multiplied <varname>PHP_FCGI_CHILDREN</varname>
```
Comment (cmb69) [inline install/unix/lighttpd-14.xml:63]:
```suggestion
   the total number of php responders will be multiplied <envar>PHP_FCGI_CHILDREN</envar>
```

File: install/unix/lighttpd-14.xml [hunk 6]
```diff
@@ -82,9 +79,9 @@ fastcgi.server = ( ".php" =>
 
   <para>
    It is possible to spawn processes without spawn-fcgi, though a bit of
-   heavy-lifting is required. Setting the PHP_FCGI_CHILDREN environment var
+   heavy-lifting is required. Setting the <varname>PHP_FCGI_CHILDREN</varname> environment var
```
Comment (cmb69) [inline install/unix/lighttpd-14.xml:82]:
```suggestion
   heavy-lifting is required. Setting the <envar>PHP_FCGI_CHILDREN</envar> environment var
```

File: install/unix/lighttpd-14.xml [hunk 7]
```diff
@@ -82,9 +79,9 @@ fastcgi.server = ( ".php" =>
...
    controls how many children PHP will spawn to handle incoming requests.
-   Setting PHP_FCGI_MAX_REQUESTS will determine how long (in requests) each
+   Setting <varname>PHP_FCGI_MAX_REQUESTS</varname> will determine how long (in requests) each
```
Comment (cmb69) [inline install/unix/lighttpd-14.xml:84]:
```suggestion
   Setting <envar>PHP_FCGI_MAX_REQUESTS</envar> will determine how long (in requests) each
```

**Review by cmb69 — COMMENTED** [review]
Thank you for the PR! I've left some sugggestions for possible improvements.

**Comment by saundefined** [comment]
Thanks!

**Review by cmb69 — APPROVED** [review]

**Pull request merged**